sql/moz-fx-data-shared-prod/acoustic_external/suppression_list_v1/query.sql

```diff
@@ -8,7 +8,7 @@ WITH base_suppression_lists AS (
     email
   -- historic upload
   FROM
-    `moz-fx-data-marketing-prod.acoustic.2024-03-20_main_suppression_list`
+    `moz-fx-data-shared-prod.acoustic_external.2024-03-20_main_suppression_list`
```

Comment:
Is this one off, or is there new data being added to the source table constantly? If not, we should probably remove the scheduling settings.

Reply:
this is a one off that is combined with the updating data